velox/connectors/hive/iceberg/tests/IcebergReadTest.cpp

```diff
@@ -240,41 +264,59 @@ TEST_F(HiveIcebergTest, positionalDeletesSingleBaseFile) {
   folly::SingletonVault::singleton()->registrationComplete();
 
   // Delete row 0, 1, 2, 3 from the first batch out of two.
-  assertPositionalDeletes({0, 1, 2, 3});
+  assertPositionalDeletes({{0, 1, 2, 3}});
   // Delete the first and last row in each batch (10000 rows per batch)
-  assertPositionalDeletes({0, 9999, 10000, 19999});
+  assertPositionalDeletes({{0, 9999, 10000, 19999}});
   // Delete several rows in the second batch (10000 rows per batch)
-  assertPositionalDeletes({10000, 10002, 19999});
+  assertPositionalDeletes({{10000, 10002, 19999}});
   // Delete random rows
-  assertPositionalDeletes(makeRandomDeleteRows(rowCount));
+  assertPositionalDeletes({makeRandomDeleteRows(rowCount)});
   // Delete 0 rows
-  assertPositionalDeletes({}, "SELECT * FROM tmp", false);
+  assertPositionalDeletes({{}}, "SELECT * FROM tmp", false);
   // Delete all rows
   assertPositionalDeletes(
-      makeSequenceRows(rowCount), "SELECT * FROM tmp WHERE 1 = 0", false);
+      {makeSequenceRows(rowCount)}, "SELECT * FROM tmp WHERE 1 = 0", false);
+  // Delete rows that don't exist
+  assertPositionalDeletes({{20000, 29999}});
+}
+
+TEST_F(HiveIcebergTest, positionalDeletesSingleBaseFileMultipleDeleteFiles) {
```

Comment:
@majetideepak this is the new test case that is added to verify the case when multiple delete files are present